content/en/docs/How To/sharding.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Sharding"
+date: 2020-10-29
+weight: 410
+description: >
+  Sharding how-to
+---
+
+Sharding is enabled in a topology by specifying a `sharding` field in the TOML
+`[output]` section. That field must be shardable though, in other words, a 
+sharding function must be associated to that field. Let's see how that works.
+
+
+#### The baker.Components structure
```

Comment:
uhm, why this paragraph?

Reply:
maybe a leftover?

Reply:
it was to introduce baker.Components very quickly (to not show the ShardingFuncs of baker.Components without context).
I've removed the paragraph though and moved the sentence into the subsequent section.
It makes more sense now :+1: